AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -11667,7 +11667,7 @@ matkahuolto.fi#%#//scriptlet('trusted-set-cookie', 'CookieConsentV2', 'YES,TOI,A
 character.ai#%#//scriptlet('set-local-storage-item', 'OPTIONAL_COOKIES_ACCEPTED_0', 'false')
 character.ai#%#//scriptlet('set-local-storage-item', 'VISITED_0', 'true')
 ilmattino.it,leggo.it,3bmeteo.com#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible button.iubenda-cs-close-btn')
-leanplayer.com,player.899thewave.fm,player.cjcbradio.com,player.max983.ca,capital.it,repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')
+leanplayer.com,player.899thewave.fm,player.cjcbradio.com,player.max983.ca,repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')
```

Comment:
it can be moved above with a comment `! Required for some sites` to avoid adding a domain to a rule that doesn't need to be applied to some site.

Reply:
What about move 
`leanplayer.com,player.899thewave.fm,player.cjcbradio.com,player.max983.ca,repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')`
 under `! iubenda CMP` section?

Then, is 
`video.repubblica.it#%#AG_onLoad(function() { setTimeout(function() { var el = document.querySelector('button.iubenda-cs-accept-btn'); if(el) el.click(); }, 5E3); });`
needed despite
`repubblica.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible .iubenda-cs-accept-btn')`
?

As concerns
`ilmattino.it,leggo.it,3bmeteo.com#%#//scriptlet('trusted-click-element', '.iubenda-cs-visible button.iubenda-cs-close-btn')`
`leggo.it` and `3bmeteo.com` can be added to
`m2o.it,deejay.it,capital.it,moto.it,automoto.it,atlantico.fr,movieplayer.it,mediaset.it,libero.it#%#//scriptlet('trusted-click-element', '.iubenda-cs-customize-btn')`
`m2o.it,deejay.it,capital.it,moto.it,automoto.it,atlantico.fr,movieplayer.it,mediaset.it,libero.it#%#//scriptlet('trusted-click-element', '#iubFooterBtn')`
while `ilmattino.it` requires more attention (also with the current filter)

Reply:
> What about move

Ok.

> Then, is

Yep)

> As concerns

Yes. But for the sites where more attention required, add a comment

Reply:
Done :)
Is it now ok?

Reply:
Yes, thanks)